flw/pom.xml

```diff
@@ -64,6 +70,18 @@
             <version>${motech.version}</version>
         </dependency>
 
+        <!-- Alerts -->

```

Comment:
not a dealbreaker but you shouldn't have to import this explicitly here since the nms pom does the global import for you.